### DISCUSSION TITLE
IndexError: Dimension out of range (expected to be in range of [-2, 1], but got 2)

### DISCUSSION THREAD

**Discussion by 40001277-FEA** [ourownstory/neural_prophet#1505] [Feature Requests & Feedback]
Hi, I am testing my neuralprophet global model to a test dataset. I trained the model with lagged regressors. But, when testing the model on the test dataset, the following error occurs:

IndexError: Dimension out of range (expected to be in range of [-2, 1], but got 2)

Test dataset:

<img width="481" alt="image" src="https://github.com/ourownstory/neural_prophet/assets/147043595/399b638f-2436-409a-ad3d-94827dbe847d">

Line of code that generates the error:

<img width="692" alt="image" src="https://github.com/ourownstory/neural_prophet/assets/147043595/036ae5d4-bc9a-4a09-9ca4-82fbb4a79c9f">

<img width="872" alt="image" src="https://github.com/ourownstory/neural_prophet/assets/147043595/7b6c00ea-ab40-4bb3-8730-7e46d2124e68">

**Comment by ourownstory** [comment]
Please feel free to open an issue and please provide the minimal code snippet for us to reproduce this issue.
